Composer/packages/server/src/models/bot/builder.ts

```diff
@@ -177,9 +217,47 @@ export class LuPublisher {
       true,
       false,
       loadResult.multiRecognizers,
-      loadResult.settings
+      loadResult.settings,
+      loadResult.crosstrainedRecognizers,
+      'crosstrained'
```

Comment:
We really need to wrap those params to a config object. Who is the owner of `.build()` interface?

Reply:
@feich-ms